model/perf.go

```diff
@@ -275,8 +278,7 @@ func (r *PerformanceResults) createFindQuery(options PerfFindOptions) map[string
 		search[bsonutil.GetDottedKeyName("info", "schema")] = options.Info.Schema
 	}
 	if len(options.Info.Tags) > 0 {
-		search[bsonutil.GetDottedKeyName("info", "tags")] =
-			db.Document{"$in": options.Info.Tags}
+		search[bsonutil.GetDottedKeyName("info", "tags")] = db.Document{"$in": options.Info.Tags}
```

Comment:
it's a little weird that we're using db.Document and bson.M in the same file. they're both map[string]interface{}